ruby/lib/minitest/queue/junit_reporter.rb

```diff
@@ -76,6 +76,8 @@ def add_tests_to(testsuites, suite, tests)
 
           testcase = testsuite.add_element('testcase', attributes)
           add_xml_message_for(testcase, test) unless test.passed?
+        rescue REXML::ParseException, RuntimeError
+          # noop
```

Comment:
Should we print any message here and let the user know we are handling the parsing exception?

Reply:
I was about to suggest the same. We should log the error.